packages/app/universal-ts-utils/src/types.ts

```diff
@@ -0,0 +1 @@
+export type NonObject = string | number | boolean | null | undefined | symbol | bigint
```

Comment:
Not sure if this is a good idea, it will exposed for FE, is it fine?

Reply:
Maybe it is better to define it internally per method, it will be duplicated but in this case I would it is fine (short piece of code)

Reply:
No strong opinion, either option is OK for me, it's a small amount of code

Reply:
Yeah ... I guess in the future we can come up with more complex shared types and we will have to decide but in this case I would prefer not exposing it

Reply:
Or maybe we can create some kind of "private" folder and adjust `export` on package.json to ignore it 🤔 

But in any case, I think we can do it independently when needed and not for this small piece of code